src/Microsoft.AspNetCore.Mvc.ViewFeatures/Rendering/HtmlHelperFormExtensions.cs

```diff
@@ -443,14 +443,14 @@ public static MvcForm BeginRouteForm(this IHtmlHelper htmlHelper, object routeVa
         /// An <see cref="object"/> that contains the parameters for a route. The parameters are retrieved through
         /// reflection by examining the properties of the <see cref="object"/>. This <see cref="object"/> is typically
         /// created using <see cref="object"/> initializer syntax. Alternatively, an
-        /// <see cref="System.Collections.Generic.IDictionary{string, object}"/> instance containing the route
+        /// <see cref="System.Collections.Generic.IDictionary{String, Object}"/> instance containing the route
         /// parameters.
         /// </param>
         /// <param name="antiforgery">
         /// If <c>true</c>, &lt;form&gt; elements will include an antiforgery token.
         /// If <c>false</c>, suppresses the generation an &lt;input&gt; of type "hidden" with an antiforgery token.
         /// If <c>null</c>, &lt;form&gt; elements will include an antiforgery token only if
-        /// <paramref name="method"/> is not <see cref="FormMethod.Get"/>.
+        /// HTTP method is not <see cref="FormMethod.Get"/>.
```

Comment:
Remove all of `if <paramref name="method"/> is not <see cref="FormMethod.Get"/>` here and in next change.